go.mod

```diff
@@ -38,12 +39,14 @@ require (
 	github.com/icrowley/fake v0.0.0-20180203215853-4178557ae428
 	github.com/jaytaylor/html2text v0.0.0-20190408195923-01ec452cbe43
 	github.com/jmoiron/sqlx v1.2.0
+	github.com/jonboulle/clockwork v0.1.0
```

Comment:
I was wondering why `github.com/jonboulle/clockwork` and `github.com/russellhaering/goxmldsig` were being added, but when I ran `go mod tidy` the `// indirect` comment is added to each. I don't think it's strictly necessary, however.

Reply:
Sometimes mine added it, other times it didn't. But I think it should be there...adding.